xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -1895,6 +1895,34 @@
       }
     }
   },
+  "PVR.AddTimer": {
+    "type": "method",
+    "description": "Adds a timer",
+    "transport": "Response",
+    "permission": "ControlPVR",
+    "params": [
+      { "name": "broadcastid", "$ref": "Library.Id", "required": true },
+      { "name": "repeating", "type": "boolean" },
+      { "name": "properties", "$ref": "PVR.Fields.Timer" }
```

Comment:
It's not obvious that these are the properties that will be returned for the created timer.